src/app/modules/registration/pages/set-avalanche-position/set-avalanche-position.page.ts

```diff
@@ -28,11 +33,15 @@ export class SetAvalanchePositionPage implements OnInit {
 
   start: L.LatLng;
   end: L.LatLng;
+  totalPolygon: IPolygon;
+  startPolygon: IPolygon;
+  endPolygon: IPolygon;
   private map: L.Map;
   private pathLine: L.Polyline;
 
   fromMarker: L.Marker;
   locationMarker: L.Marker;
+  locationPolygon = new Subject<IPolygon>();
```

Comment:
Obs: Flisespikkeri! Det blir bittelitt rart med navngiving når IPolygon har et felt som heter polygon. Eksempel: this.totalPolygon.polygon... klinger litt rart. Kunne du gjort noe med navn på interface eller felt, evt. variable som gjør dette mer elegant?

Reply:
Kommer ikke på noen synonym til polygon, så vil la denne være som den er.